src/components/global/Playground/index.tsx

```diff
@@ -25,6 +34,34 @@ export default function Playground() {
     copyButton.click();
   }
 
+  function openEditor(event) {
+    const button = event.target.closest('button');
+    // Outer text includes line breaks `\n` to maintain code formatting in editor examples.
+    const codeBlock = button.closest('.playground').querySelector('.code-block pre code')?.outerText ?? '';
+    const editorOptions = {
+      title,
+      description,
+    };
+
+    switch (usageTarget) {
+      case UsageTarget.Angular:
+        openAngularEditor(codeBlock, editorOptions);
+        break;
+      case UsageTarget.Html:
+        openHtmlEditor(codeBlock, editorOptions);
+        break;
+      case UsageTarget.React:
+        openReactEditor(codeBlock, editorOptions);
+        break;
+      case UsageTarget.Vue:
+        openVueEditor(codeBlock, editorOptions);
+        break;
+    }
+  }
+
+  // TODO: This variable will be used for generating the buttons to change the code language/framework.
+  const languages = UsageTargetList.filter((lang) => Object.keys(codeSnippet).includes(lang.toLowerCase()));
```

Comment:
Since the default value for `codeSnippet` includes all language keys from the start, the `includes` check here is redundant (will always match all four languages). Should it be checking to make sure the value for the key is truthy instead?

Reply:
Good catch. I'm going to remove this for now. I think it should be handled as the scope for adding the buttons.